app/src/main/java/org/thoughtcrime/securesms/conversation/start/newmessage/NewMessage.kt

```diff
@@ -142,8 +144,10 @@ private fun EnterAccountId(
                     SessionOutlinedTextField(
                         text = state.newMessageIdOrOns,
                         modifier = Modifier
-                            .padding(horizontal = LocalDimensions.current.spacing),
-                        contentDescription = "Session id input box",
+                            .padding(horizontal = LocalDimensions.current.spacing)
+                            .semantics {
+                                contentDescription = "Session id input box"
```

Comment:
I'm not  sure why this is here, even though the old code already had it. But we shouldn't hardcode strings anywhere, especially something that should be translated.
If this is a qa tag then it should:
- be moved to the appropriate string file for qa tags
- be using the new `qaTag` modifier instead of misusing content description which should be reserved for accessibility